inst/htmljs/animint.js

```diff
@@ -2392,11 +2392,11 @@ var animint = function (to_select, json_file) {
       .attr("class", "table_selector_widgets")
     ;
     var selector_first_tr = selector_table.append("tr");
-    setTimeout(() => {
-      const selector_height = selector_table[0][0].clientHeight;
-      selector_table
-        .style('height', `${selector_height}px`);
-    }, 0);
+    // setTimeout(() => {
```

Comment:
since the fix is commented, and the tests are still passing, this seems to imply that the test is not sufficient to capture the bug, is that right? @Faye-yufan

Reply:
Correct, this version of the test cannot capture the bug, I'm thinking of a new way to replicate the bug.